manifests/mod/event.pp

```diff
@@ -99,16 +99,9 @@
     notify  => Class['apache::service'],
   }
 
-  case $facts['os']['family'] {
-    'redhat': {
-      if versioncmp($_apache_version, '2.4') >= 0 {
-        apache::mpm { 'event':
-          apache_version => $_apache_version,
-        }
-      }
-    }
-    'debian', 'freebsd' : {
-      apache::mpm { 'event':
+  case $::osfamily {
```

Comment:
please don't reference facts as top scope variables als also please don't use legacy facts:
```suggestion
  case $facts['os']['family'] {
```

Reply:
when this case has no just a single match, maybe `if $facts['os']['family'] in ['Debian', 'FreeeBSD', 'RedHat']`

Reply:
Yeh, only just noticed this. Missed that it was overwritten by the rebasse